src/components/pages/TeachersPage/index.tsx

```diff
@@ -156,6 +156,42 @@ export default function TeachersPage({ classroomName }: ClassroomProps) {
           {showDisplayedChat()}
         </Grid>
       </Grid>
+      <Grid
```

Comment:
I feel the multichat display should be its own component. Not sure what the best name for it is. Maybe `<AllStudentChatsDisplay>`?